hack/update-manifests.sh

```diff
@@ -35,17 +37,22 @@ cd ${SRCROOT}/manifests/base && $KUSTOMIZE edit set image quay.io/argoproj/argoc
 cd ${SRCROOT}/manifests/ha/base && $KUSTOMIZE edit set image quay.io/argoproj/argocd=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG}
 cd ${SRCROOT}/manifests/core-install && $KUSTOMIZE edit set image quay.io/argoproj/argocd=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG}
 
-echo "${AUTOGENMSG}" > "${SRCROOT}/manifests/install.yaml"
+echo -n "" > "${SRCROOT}/manifests/install.yaml"
 $KUSTOMIZE build "${SRCROOT}/manifests/cluster-install" >> "${SRCROOT}/manifests/install.yaml"
+go run $HACK_IMAGEPULLPOLICY -manifest=${SRCROOT}/manifests/install.yaml -image=${IMAGE_NAMESPACE}/argocd:${IMAGE_TAG} -image-pull-policy=$IMAGE_PULL_POLICY
```

Comment:
It would be preferable to keep this script Kustomize centric instead of mixing Go code in it.